(DOCSP-23676): Source constant, admonition, term cleanup (#2034)
mongodb/docs-realm: source/sdk/node/advanced/multi-user-applications.txt

- **Logged Out:** any user that authenticated on the device but
  has since logged out or had their session revoked.

> The following diagram shows how users within a Realm client app

Author: ```suggestion
The following diagram shows how users within an App Services client app
```
